cmd/xl-storage.go

```diff
@@ -2077,6 +2077,16 @@ func (s *xlStorage) deleteFile(basePath, deletePath string, recursive, force boo
 			if isObjectDir {
 				return errFileNotFound
 			}
+			// if we have .DS_Store from macOS, it will copy to any where
+			storeFilePath := pathJoin(deletePath, ".DS_Store")
```

Comment:
Only enable when `if runtime.GOOS == "darwin" {` - this will be pointless in most installs.

Reply:
It needs to be changed slightly.